src/config/config_type.rs

```diff
@@ -400,6 +409,17 @@ macro_rules! create_config {
                 }
             }
 
+            fn set_fn_args_layout(&mut self) {
+                if self.was_set().fn_args_layout() {
+                    eprintln!(
+                        "Warning: the `fn_args_layout` option was renamed to `fn_params_layout`."
```

Comment:
Can we also mention that the old option has been deprecated, similar to what we do for other such renames?

Reply:
Absolutely! I'll make sure to include those details.